github/sendgrid/pr-thanks-by-sendgrid/rust/src/pr-thanks-by-sendgrid.rs

```diff
@@ -0,0 +1,50 @@
+use serde_json::json;
+use flows_connector_dsi::{github::inbound};
+use wasmedge_bindgen_macro::*;
+
+#[wasmedge_bindgen]
+pub fn run(s: String) -> Result<String, String> {
+    #[allow(unused_imports)]
+    use wasmedge_bindgen;
+    _run(s)
+}
+
+fn _run(s: String) -> Result<String, String> {
+    let payload = inbound(s)?;
+    
+    payload.get_pull_request()?;
+
+    if payload.get_action()? != "opened" {
+        return Ok(String::new());
+    }
+    
+    let subject = "Thank you for contributing to this repository";
+    let content = format!(
+            r#"
+Hi {}, <br/>
+
+Welcome to the {} community, thank you for your contribution!
+                        "#,
+            payload.sender.login, payload.repository.unwrap().full_name
+        );
+    
+
+    Ok(json!([{
```

Comment:
Actually you can use `flows_connector_dsi::sendgrid::outbound` here if you like😂